src/javaRosa.js

```diff
@@ -941,7 +941,7 @@ define([
         widget.setItextValue = function (value) {
             var itextItem = widget.getItextItem();
             if (itextItem) {
-                if (!value) {
+                if (!value && !mug.options.canHaveBlankLabel) {
                     value = widget.getPlaceholder();
```

Comment:
Does this mean that all group question types (Group, Repeat, Field List) will not use the question ID as default label?